lib/streamlit/components/v1/components.py

```diff
@@ -184,6 +185,26 @@ def marshall_element_args():
 
             if key is None:
                 marshall_element_args()
```

Comment:
nit: need to update the comment above the definition of `marshall_element_args` to reflect what happens in the new world (it still refers to hashing protobufs)